opt-env-conf/src/OptEnvConf/Doc.hs

```diff
@@ -321,7 +321,9 @@ renderManPage progname version progDesc docs =
               [".Sh ", "SYNOPSIS"],
               renderShortOptDocs progname optDocs,
               [".Sh ", "SETTINGS"],
-              renderSetDocs docs
+              renderSetDocs docs,
+              [".Sh ", "COMMANDS"],
+              renderCommandDocs docs
```

Comment:
I like the idea of having a `COMMANDS` section, but:

1. Not if there are no commands
2. They should list the per-command options there